tests/test_testing.py

```diff
@@ -103,6 +103,20 @@ def on_post(self, req, resp):
     )
     assert result.text == falcon.MEDIA_JSON
 
+    result = testing.simulate_post(app, '/', msgpack={})
+    assert result.text == falcon.MEDIA_MSGPACK
+
+    result = testing.simulate_post(app, '/', msgpack={}, content_type=falcon.MEDIA_HTML)
+    assert result.text == falcon.MEDIA_MSGPACK
+
+    result = testing.simulate_post(app, '/', msgpack={}, headers=headers)
+    assert result.text == falcon.MEDIA_MSGPACK
+
+    result = testing.simulate_post(
+        app, '/', msgpack={}, headers=headers, content_type=falcon.MEDIA_HTML
+    )
+    assert result.text == falcon.MEDIA_MSGPACK
```

Comment:
there are ok, but the existing test were not that great. like we didn't actually check if the body was correctly serialized.

We could probably do better, but it's fine not to do these in this PR